runtime/primitives/src/currency.rs

```diff
@@ -112,11 +113,41 @@ impl CurrencyId {
 		pub const KSM: CurrencyId = CurrencyId(4);
 		pub const PBLO: CurrencyId = CurrencyId(5);
 
-		/// Karura stable coin (Karura Dollar), not native.
+		// Non-Native Tokens (101 - 1000)
+		/// Karura stable coin(Karura Dollar), not native.
 		#[allow(non_upper_case_globals)]
 		pub const kUSD: CurrencyId = CurrencyId(129);
 		pub const USDT: CurrencyId = CurrencyId(130);
 		pub const USDC: CurrencyId = CurrencyId(131);
+		/// Wrapped BTC
+		#[allow(non_upper_case_globals)]
+		pub const wBTC: CurrencyId = CurrencyId(132);
+		/// Wrapped ETH
+		#[allow(non_upper_case_globals)]
+		pub const wETH: CurrencyId = CurrencyId(133);
+		/// Acala Dollar
+		#[allow(non_upper_case_globals)]
+		pub const aUSD: CurrencyId = CurrencyId(134);
+
+		// Staked asset xTokens (1001 - 2000)
```

Comment:
This and the staking NFTs grow in parallel so it makes sense to allow both of them to grow at the same rate and size. May be let's set aside 100k ids for both? (Being over optimistic I know ;)

Reply:
These ranges are only for the preset fNFT IDs that are declared in our runtime code. The ID range for user-created ones have a massive range declared in Currency factory.
Do we still want the 100k range for the hard-coded ones?